src/index.ts

```diff
@@ -1837,6 +1839,20 @@ async function getLinkDetails({ link, provider }: interfaces.IGetLinkDetailsPara
 	}
 }
 
+async function resolveToENSName({
```

Comment:
I guess seeing how simple this function is, does it even make sense to have it? It's just a wrapper around ethers function.

Reply:
Although the complexity here might increase if we're resolving other namesystems too (e.g. unstoppable domains, beam names, coinbase ids, etc)

Reply:
Yep, this was my thinking too. The integrator does not have to worry about what nameSystem & provider to use, just passin the address and get the name back